Cargo.toml

```diff
@@ -32,7 +32,7 @@ futures = { version = "^0.3" }
 futures-util = { version = "^0.3" }
 rust_decimal = { version = "^1", optional = true }
 sea-orm-macros = { version = "^0.1.1", optional = true }
-sea-query = { version = "^0.15", features = ["thread-safe"] }
+sea-query = { version = "^0.15", git = "https://github.com/SeaQL/sea-query.git", features = ["thread-safe"] }
```

Comment:
Use sea-query on crates.io after new version published

Reply:
Yea 0.16 is coming

Reply:
[0.16.0](https://github.com/SeaQL/sea-query/releases/tag/0.16.0)

Reply:
![image](https://user-images.githubusercontent.com/1782664/131806083-c3d38f80-c557-47cf-8ac2-4b7023cea879.png)